libs/domain/search/facet/facet.component.spec.ts

```diff
@@ -256,4 +325,154 @@ describe('SearchFacetComponent', () => {
       expect(callback.mock.calls[0][0].detail).toStrictEqual(detail);
     });
   });
+
+  describe('facet values', () => {
+    let listItems: NodeListOf<HTMLLIElement>;
+
+    beforeEach(async () => {
+      element = await fixture(
+        html` <oryx-search-facet
+          name="Mock"
+          .renderLimit="${(mockFacet.values as FacetValue[]).length}"
+        ></oryx-search-facet>`
+      );
+
+      listItems = element.renderRoot.querySelectorAll('li');
+    });
+
+    it('should render input value', () => {
+      (mockFacet.values as FacetValue[]).forEach((value, index) => {
+        expect(listItems[index].querySelector('input')?.value).toBe(
+          value.value
+        );
+      });
+    });
+
+    it('should render the label', () => {
+      (mockFacet.values as FacetValue[]).forEach((value, index) => {
+        expect(
+          listItems[index].querySelector('input + div span')?.textContent
+        ).toContain(value.name);
+      });
+    });
+
+    it('should render the count number for facet values which count property is greater than 0', () => {
+      const values = (mockFacet.values as FacetValue[]).filter(
+        (val) => val.count > 0
+      );
+
+      const facetOptions = Array.from(listItems).filter(
+        (item) =>
+          item.querySelectorAll('input + div span').length === 2 &&
+          item.querySelector('input + div span:last-child')?.textContent
+      );
+
+      expect(values.length).toBe(facetOptions.length);
```

Comment:
You should know exact expectation, why do you need to create `facetOptions` array? What are you checking in this test? It is not related to the component

Reply:
`facetOptions` is an array of `li` elements that have a counter. This test checks that the number of `li` elements with counters is the same as the number of values with `count > 0`.